subcommittee/coding-guidelines/initiatives/safe-use-of-unsafe-guidelines/learn_unsafe_rust/src/core_unsafety/intrinsics.md

```diff
@@ -0,0 +1,13 @@
+
+# Intrinsics
+
+Rust provides a set of compiler intrinsics as part of `core` and `std` respectively (https://doc.rust-lang.org/std/intrinsics/index.html).
+
+Most of them are marked as experimental, unstable and useable in nightly versions of the compiler.
+
+For many, there is already stabilized API available.
```

Comment:
"stabilized API" could be expanded a bit more, such that a reader doesn't start looking for library crates to use these intrinsics.

```suggestion
Many intrinsics are accessible though a stabilized API located elsewhere in the Standard or Core Library. The documentation for each intrinsic typically references the stabilized API.
```